inst/rmarkdown/templates/mortality/skeleton/skeleton.Rmd

```diff
@@ -1014,24 +1180,24 @@ You could similarly repeat this looking at distributions by sex.
 Note that these p-values are just indicative, and a descriptive discussion (or
 visualisation with age-pyramids below) of the distributions in your study sample 
 compared to the source population is more
-important that the binomial test itself. This is because increasing sample size
+important than the binomial test itself. This is because increasing sample size
 will more often than not lead to differences that may be irrelevant after weighting
 your data. 
 
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~-->
 
 ```{r descriptive_sampling_bias, warning = FALSE}
 ## counts and props of the study population
-ag <- tab_linelist(study_data_cleaned, "age_group") %>%
-  mutate(n_total = sum(n)) %>%
-  select(-variable) %>%
-  rename(age_group = value) %>%
-  mutate(age_group = fct_inorder(age_group))
+ag <- tabyl(study_data_cleaned, age_group, show_na = FALSE) %>% 
```

Comment:
@pbkeating here using tabyl much easier because the whole chunk is generally quite dense. 
But at the moment we dont have a work around becuase need to pull population counts in so cant do directly with gtsummary. 
Doing this basic counts with gtsummary and then pulling in to a tibble for meging with population counts is really dense, cus is all packaged in lists